tools/coqdep_common.ml

```diff
@@ -11,6 +11,8 @@
 open Format
 open Unix
 open Coqdep_lexer
+(* The whole Minisys module used to be required due to bootstrapping,
+   thanks to dune it is not needed anymore and we could just remove it. *)
```

Comment:
Why don't we remove it then? Or do you prefer to do that in the next PR?

Reply:
Yup, I guess I will take care of this one in #14059